getting_started.md

```diff
@@ -0,0 +1,104 @@
+
+![Scuttlebutt Logo](assets/hermies-300x300.png)
+
+Ok, you've been hearing about [Scuttlebutt](https://scuttlebutt.nz)  and decided that "Yes, I do want to join an amazing social network with lots of good people that no company can control and also happens to also work offline. 
+
+Here's a quick overview with the basics you need to know.
+
+First off Scuttlebutt is a protocol on which many different kinds of apps can be built. 
+
+As for the social network, there are many clients, just like there are many Twitter clients. It doesn't really matter which one you use. They're all talking on the same network. 
+
+Patchwork is the most polished client and it doesn't require you to know any geeky stuff. The only caveat is that it just wants to run on one computer, and your identity is tied to the files on the computer you install it on. There are ways around this but it's beyond the scope of this.
+
+![screenshot of Patchwork](assets/patchwork.jpg)
+
+
+## Step 0
+If you haven't done so already go to [the Scuttlebutt Home Page](README.md) and watch the video. Then come back. 
+
+## Step 1
+[Download Patchwork](http://dinosaur.is/patchwork-downloader/), and install it like any other app. 
+ 
+## Step 2
+Boot it up and set up your profile. 
+
+![patchwork new profile screenshot](assets/patchwork-first-boot.png)
+
+It doesn't matter what name you put. It can be your real name. It can be a nickname. Like the real world it's entirely possible for multiple people to have the same name. Your friends can give you nicknames too. 
+
+You don't need to upload an image, but I recommend that you do, because it's much easier to figure out who is talking when there's a visual, and the defaults are really generic. Some people use a photo of themselves. Some people use a random avatar they like, just like Twitter.
+
+The text area, like every text area in Patchwork uses [markdown](http://commonmark.org/help/) formatting.
```

Comment:
might be good to list some special [Scuttlebutt markdown (`ssb-markdown`)](https://github.com/ssbc/ssb-markdown) things?

Reply:
yeah, i _want_ to do that, and i intend to change it eventually, but so far the only docs I've found are _terrible_ looking and absolutely not something you want to show a new user unfamiliar with markdown.  Maybe there's a pretty version with visualized formatting. For example: actually showing a link not just saying "`[link text](link_url)` makes a link". If so i haven't found it.  I figured do this first, then tackle those docs, then update this link.